test/VoteManager.js

```diff
@@ -87,6 +87,11 @@ describe('VoteManager', function () {
         await stakeManager.connect(signers[4]).stake(epoch, tokenAmount('19000'));
       });
 
+      it('should not be able to initialize contracts if they are already initialized', async function () {
+        const tx = Promise.all(await initializeContracts());
```

Comment:
Promise.all() is not required here.

Reply:
ok so I will call initialize only on one contract (votemanager) which is already initialized @hrishikeshio